# unfortunately, vienna-rna's build system is a little broken and sometimes detects
# gd in RNAforested but then realizes in g2 that it can't actually build it 
# resulting in unresolved symbols.

diff -Naur ViennaRNA-1.7.2/RNAforester/configure.in ViennaRNA-1.7.2.new/RNAforester/configure.in
--- ViennaRNA-1.7.2/RNAforester/configure.in	2006-01-19 10:26:09.000000000 -0500
+++ ViennaRNA-1.7.2.new/RNAforester/configure.in	2008-10-06 14:08:56.000000000 -0400
@@ -31,7 +31,7 @@
 CPPFLAGS=["-Ig2-0.70/src/ -Lg2-0.70/ ${CPPFLAGS}"]	
 
 			    
-AC_CHECK_LIB(gd,gdImageLine,,)
+dnl AC_CHECK_LIB(gd,gdImageLine,,)
 		  
 dnl Checks for header files.
 AC_HEADER_STDC
